libs/filamat/src/GLSLPostProcessor.cpp

```diff
@@ -590,6 +611,21 @@ void GLSLPostProcessor::optimizeSpirv(OptimizerPtr optimizer, SpirvBlob& spirv)
     remapper.remap(spirv, spv::spirvbin_base_t::DCE_ALL);
 }
 
+void GLSLPostProcessor::fixupClipDistance(
+        SpirvBlob& spirv, GLSLPostProcessor::Config const& config) const {
+    if (!config.usesClipDistance) {
+        return;
+    }
+    SpirvTools tools(SPV_ENV_UNIVERSAL_1_3);
+    std::string disassembly;
+    tools.Disassemble(spirv, &disassembly);
```

Comment:
can this fails? (e.g. if in the future UNIVERSAL_1_3 is not the right constant).

Reply:
It can, I'll add an assertion and comment that it should match the version we use here:

https://github.com/google/filament/blob/731dd761d9fb6b97c557d4ba6c8e77a9b0babca8/libs/filamat/src/sca/GLSLTools.cpp#L443